sandpack-react/src/components/CodeEditor/__snapshots__/useSyntaxHighlight.test.tsx.snap

```diff
@@ -1456,6 +1456,21 @@ Array [
 
 exports[`useSyntaxHighlight renders a markdown block 1`] = `null`;
 
+exports[`useSyntaxHighlight renders a python block 1`] = `
+Array [
+  "import os
+
+api_token = os.environ",
+  <span
+    className="ͼo"
+  >
+    ['API_TOKEN']
+  </span>,
+]
+`;
+
+exports[`useSyntaxHighlight renders a shell block 1`] = `null`;
```

Comment:
Interesting why the output is null, could you take a look at this?

Reply:
This is odd - it looks like it's stumbling on the provided code sample? If I change the mocked "shell" code to the Python code it works fine.

Reply:
Was exploring this a bit more, and it looks like this occurs with Markdown as well: https://github.com/codesandbox/sandpack/blob/main/sandpack-react/src/components/CodeEditor/__snapshots__/useSyntaxHighlight.test.tsx.snap#L1457